prow/pod-utils/gcs/upload.go

```diff
@@ -35,29 +35,45 @@ type UploadFunc func(obj *storage.ObjectHandle) error
 // Upload uploads all of the data in the
 // uploadTargets map to GCS in parallel. The map is
 // keyed on GCS path under the bucket
+// 3 attempts will be made to have an error free upload
+const uploadRetries = 3
```

Comment:
I think this will push the doc for the Upload method onto this const in godoc -- can we format as:

```go
const uploadRetries = 3

// Upload uploads all of the data in the ...
```